### DISCUSSION TITLE
Longterm caching with Polars

### DISCUSSION THREAD

**Discussion by insanpraja** [getdozer/dozer#1764] [Feature Requests]
![image](https://github.com/getdozer/dozer/assets/73377524/cda903a1-a903-4125-b7e0-5b5735e3d71a)

We were trying to do this with Dozer, but we can't find ways to tell Dozer to read Polars when the query is, let's say, not exceeding 30 days/1 month, and read from DB (PgSQL/MySQL) if it's older than Polars cache. 

Any suggestions?

**Comment by chubei** [comment]
Hi insanpraja, can you explain why you need Dozer to only read data within a month? I can't get it from the diagram you posted.

**Reply by insanpraja** [reply]
Hi @chubei, our intention is to cache 30 days of data in polars, and the rest of the data stay in MySQL database.

Thanks.

**Reply by chubei** [reply]
In this case, Dozer can directly connect to MySQL. It's not necessary to go though the polars cache.

Dozer does one snapshot of your database when it's first started, then it listens to CDC events to keep data up to date.

**Reply by insanpraja** [reply]
Thanks for insight!
